src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -105,13 +105,14 @@ def run(self, revision):
         ] + list(revision.files)
         logger.info('Running static-analysis', cmd=' '.join(cmd))
 
-        # Run command
+        # Run command, without checking its exit code as clang-tidy 7+
+        # exits with an error code when finding errors (which we want to report !)
         try:
```

Comment:
Is the exception handling still necessary? Will this change hide other possible errors?